test/java/ch/ehi/ili2db/FetchTest.java

```diff
@@ -0,0 +1,114 @@
+package ch.ehi.ili2db;
+
+import ch.ehi.ili2db.base.Ili2db;
+import ch.ehi.ili2db.gui.Config;
+import ch.interlis.iom.IomObject;
+import ch.interlis.iom_j.xtf.XtfReader;
+import ch.interlis.iox.EndBasketEvent;
+import ch.interlis.iox.EndTransferEvent;
+import ch.interlis.iox.IoxEvent;
+import ch.interlis.iox.ObjectEvent;
+import ch.interlis.iox.StartBasketEvent;
+import ch.interlis.iox.StartTransferEvent;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.HashMap;
+
+public abstract class FetchTest {
+
+    protected static final String TEST_OUT="test/data/FetchSize/";
+    protected AbstractTestSetup setup=createTestSetup();
+    protected abstract AbstractTestSetup createTestSetup() ;
+
+    @Test
+    public void exportXtf() throws Exception
+    {
+        {
+            importXtf();
```

Comment:
Wahrscheinlich macht es mehr Sinn, hier für diesen Test nur ein Schema-Import zu machen (importIli()) und dann in der Tabelle mit SQL Records zu erzeugen und dann den Export  zu machen.